### PR TITLE
VDS-855: Add 'View purchased products' block to category sidebar

### PR DESCRIPTION
**Pull request by ilyawzrd**
Снимок экрана 2021-02-09 в 15 24 31

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/vc-demo-theme-b2b-1.10.0-pr-175.zip

### CODE REVIEW

**Review by pushnitsa — APPROVED** [review]

**Review by trueboroda — APPROVED** [review]

File: snippets/collection/sidebar.liquid [hunk 1]
```diff
@@ -2,7 +2,9 @@
     {% include 'collection/sidebar/applied-filters' %}
     {% include 'collection/sidebar/keyword' %}
     {% if customer and customer.is_registered_user %}
-        {% include 'collection/sidebar/previously-purchased' %}
+        {% unless customer.is_first_time_buyer %}
```
Comment (pushnitsa) [inline snippets/collection/sidebar.liquid:5]: This statement could be simplified and unioned with if above like that: customer.is_first_time_buyer != true

**Review by pushnitsa — CHANGES_REQUESTED** [review]

File: snippets/collection/sidebar.liquid (hunk 1 above)
Comment (ilyawzrd) [reply snippets/collection/sidebar.liquid:5]: i think it's just a preference. 'unless' works the same as '!="
https://github.com/Shopify/liquid/issues/138

Comment (pushnitsa) [reply snippets/collection/sidebar.liquid:5]: Yes, but instead of two lines with the statement, we could have just one.

Comment (ilyawzrd) [reply snippets/collection/sidebar.liquid]: done

**Review by pushnitsa — APPROVED** [review]

**Pull request merged**